tests/mappers/test_distcp_mapper.py

```diff
@@ -164,12 +165,12 @@ def test_task_and_relations_no_prepare(self):
 
 
 def _get_distcp_mapper(distcp_node: Element, job_properties: Dict[str, str], config: Dict[str, str]):
-    name = "distcp"
+    _name = "distcp"
```

Comment:
We don't use a private variables in this project, because variables in most cases is private. I prefer to keep original variable name. What's benefits of this change?